diagnostic.md

```diff
@@ -150,7 +170,11 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+Div 1 = top right
+Div 2 = top left
+Div 3 = top center
+Div 4 = middle left
+Div 5 = middle center below box 3
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5